hasher-matcher-actioner/webapp/src/Signals.jsx

```diff
@@ -45,41 +45,39 @@ export default function Signals() {
 
 function SignalSummary({summary}) {
   return (
-    <>
-      <Col md="12" className="mb-4">
-        <Card>
-          <Card.Header as="h4" className="text-white bg-success">
-            {summary.name}
-          </Card.Header>
-          <Card.Body className="py-0">
-            <Table className="mb-0">
-              <thead>
+    <Col key={summary.name} md="8" className="mb-4">
+      <Card>
+        <Card.Header as="h4" className="text-white bg-primary">
+          Dataset: {summary.name}
+        </Card.Header>
+        <Card.Body className="py-0">
+          <Table className="mb-0">
+            <thead>
+              <tr>
+                <th>Signal Type</th>
+                <th>Number of Signals</th>
+              </tr>
+            </thead>
+            <tbody>
+              {summary.signals !== null && summary.signals.length ? (
+                summary.signals.map(signal => (
+                  <tr key={signal}>
```

Comment:
I took a quick look and it does sound like this should be a string, not an object. Behind the scenes React is probably doing a "to string" on each `signal` as its mapped. From the docs:

The best way to pick a key is to use a string that uniquely identifies a list item among its siblings.

https://reactjs.org/docs/lists-and-keys.html#keys

So it's probably better to pick something that's a bit more predictable and concise if possible. JavaScript toString() creates a big chunk of text? (I don't know what the javascript tostring output on `signal` will be, for example?